### DISCUSSION TITLE
Are you using node-ble in your project?

### DISCUSSION THREAD

**Discussion by chrvadala** [chrvadala/node-ble#16] [Show and tell]
Are you using **node-ble** in your project? :grey_question: 

Share your project and show how you used **node-ble** in your code. :technologist: :woman_technologist:

:star: :star: :star: :star: :star:

**Comment by rabee05** [comment]
I have been using since last week. I definitely like the idea of using d-bus. I totally understand d-bus is the recommended way, but I noticed it is very slow compared to HCI socket. Bluez has gone their major changes, but not sure of dbus performance will be improved over time. Overall great work and priceless effort.
